frontend/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>marsRover</title>
```

Comment:
What's the <title> tag used for?

Reply:
To add a label to the tab for this page in the browser.

Reply:
![image](https://user-images.githubusercontent.com/44932469/98805520-98b46280-240f-11eb-84cc-c5391ce71872.png)

Reply:
Yesss. In that case, since you've _chosen_ to format 'mars rover' in that way, that looks fine :)